open-ce/utils.py

```diff
@@ -82,6 +82,30 @@ class Argument(Enum):
                                         default=DEFAULT_BUILD_TYPES,
                                         help='Comma delimited list of build types, such as "cpu" or "cuda".'))
 
+    DOCKER_BUILD = (lambda parser: parser.add_argument(
+                                        '--docker_build',
+                                        action='store_true',
+                                        help="Perform a build within a docker container. "
+                                             "NOTE: When the --docker_build flag is used, all arguments with paths "
+                                             "should be relative to the directory containing open-ce. Only files "
+                                             "within the open-ce directory and local_files will be visible at "
```

Comment:
I think this should be "Only files and folders within the current working directory will be visible"  unless you're forcing them to be run at a specific level.